kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -434,7 +436,15 @@ def exec_show_kcfg(
         exclude_claim_labels=exclude_claim_labels,
     )
 
-    proof_show = APRProofShow(kevm, node_printer=KEVMNodePrinter(kevm))
+    node_printer: NodePrinter
+    if type(proof) is APRBMCProof:
+        node_printer = KEVMAPRBMCNodePrinter(kevm, proof)
+    elif type(proof) is APRProof:
+        node_printer = KEVMAPRNodePrinter(kevm, proof)
+    else:
+        raise ValueError(f'Cannot build shower for proof type: {type(proof)}')
+    proof_show = APRProofShow(kevm, node_printer=node_printer)
```

Comment:
This logic is duplicated a few times. Can it be extracted into a new function?

Reply:
Done.